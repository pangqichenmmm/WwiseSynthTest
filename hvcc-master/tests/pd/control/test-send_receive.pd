#N canvas 556 166 227 163 10;
#X obj 12 13 r hello;
#X obj 12 38 print hello;
#X obj 14 63 r world;
#X obj 14 88 print world;
#X obj 67 13 r hello;
#X obj 119 13 loadbang;
#X obj 138 81 send hello;
#N canvas 0 23 114 71 subpatch 0;
#X obj 13 38 send world;
#X obj 13 14 inlet;
#X connect 1 0 0 0;
#X restore 119 102 pd subpatch;
#X obj 158 59 s;
#X obj 119 36 t a a a;
#X obj 14 111 r;
#X obj 14 135 print noname;
#X connect 0 0 1 0;
#X connect 2 0 3 0;
#X connect 4 0 1 0;
#X connect 5 0 9 0;
#X connect 9 0 7 0;
#X connect 9 1 6 0;
#X connect 9 2 8 0;
#X connect 10 0 11 0;
