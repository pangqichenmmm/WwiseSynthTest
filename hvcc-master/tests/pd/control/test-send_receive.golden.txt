[@ 0.000] noname: bang
[@ 0.000] hello: bang
[@ 0.000] hello: bang
[@ 0.000] world: bang
